src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 
 ## Version 7.0.0
 * [Breaking Change] Renamed the property `ResouceType` of `ASRVaultSettings` to `ResourceType`. 
+* Added support for enabling Disk access settings for managed VM restores.
```

Comment:
move this line to line 21 under upcoming release

Reply:
updated